contracts/tokens/NFToken.sol

```diff
@@ -350,7 +350,7 @@ contract NFToken is
   {
     address from = idToOwner[_tokenId];
 
-    clearApproval(from, _tokenId);
+    delete idToApprovals[_tokenId];
```

Comment:
How about `if (idToApprovals[_tokenId] != 0) { delete idToApprovals[_tokenId]; }`?

Reply:
I did some gas estimate tests and the results are:
if statement: 315 
delete: 5320

So this change makes sense. Will update.